PhysicsTools/JetMCAlgos/plugins/HadronAndPartonSelector.cc

```diff
@@ -254,7 +254,7 @@ HadronAndPartonSelector::produce(edm::Event& iEvent, const edm::EventSetup& iSet
      partonSelector_->run(particles,partons);
      for(reco::GenParticleCollection::const_iterator it = particles->begin(); it != particles->end(); ++it)
      {
-       if( !(it->status()==3 || (( partonMode_=="Pythia8" ) && (it->status()==23)))) continue;
+      // if( !(it->status()==3 || (( partonMode_=="Pythia8" ) && (it->status()==23)))) continue;
```

Comment:
it looks like this part will be modified to become configurable.
If it would not, the following applies:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant